draft-ietf-quic-multipath.md

```diff
@@ -430,25 +431,22 @@ Long delays may cause the peer to repeat the PATH_CHALLENGE and eventually
 send a PATH_ABANDON, in which case the procedures specified in
 Section {{path-close}} apply.
 
-PATH_ACK frames (defined in {{mp-ack-frame}}) can be returned on any path.
+Any frames can be sent on any open path.
```

Comment:
Previous version is better. PATH_CHALLENGE and PATH_RESPONSE frames cannot be "sent on any open path".

Reply:
The point is PATH_CHALLENGE and PATH_RESPONSE frames are not special. There are no restrictions for any frame on which path to sent. We don't say this anywhere. I think it good to say that once somewhere.

Reply:
They are special. You cannot send a challenge or a response for path 1 on path2...

Reply:
I think we should keep the old text, for the same reason that we have text stating that "PATH_ABANDON frames can be sent on any path, not only on the path that is intended to be closed." It is important to assert that loudly, so implementers do not start assuming that PATH_ACK can only be sent on the path on which the acked packets were sent.

Reply:
Okay you are right. Reverting it.

Reply:
```suggestion
PATH_ACK frames (see {{mp-ack-frame}}) can be returned on any path.
```